addons/mass_mailing_sms/models/res_users.py

```diff
@@ -49,7 +51,7 @@ def systray_get_activities(self):
                         icon = module and modules.module.get_module_icon(module)
                         res_ids = set()
                         user_activities[act['mailing_type']] = {
-                            'id': self.env['ir.model']._get('mailing.mailing').id,
+                            'id': len(activities) + id_offset,
```

Comment:
Same question as in the master PR, can't the client rely on the index of the activityGroups if it is only used for the owl foreach?